src/accounting.rs

```diff
@@ -1,56 +1,63 @@
-use crate::constants::{
-    CANISTER_OVERHEAD, COLLATERAL_CYCLES_PER_NODE, HTTP_OUTCALL_REQUEST_BASE_COST,
-    HTTP_OUTCALL_REQUEST_COST_PER_BYTE, HTTP_OUTCALL_REQUEST_PER_NODE_COST,
-    HTTP_OUTCALL_RESPONSE_COST_PER_BYTE, INGRESS_MESSAGE_BYTE_RECEIVED_COST,
-    INGRESS_MESSAGE_RECEIVED_COST, INGRESS_OVERHEAD_BYTES, NODES_IN_SUBNET, RPC_URL_COST_BYTES,
-};
+use crate::constants::{COLLATERAL_CYCLES_PER_NODE, INGRESS_OVERHEAD_BYTES, RPC_URL_COST_BYTES};
 
 /// Calculates the cost of sending a JSON-RPC request using HTTP outcalls.
-pub fn get_http_request_cost(payload_size_bytes: u64, max_response_bytes: u64) -> u128 {
-    let nodes_in_subnet = NODES_IN_SUBNET as u128;
+/// See https://internetcomputer.org/docs/current/developer-docs/gas-cost/#https-outcalls
+pub fn get_http_request_cost(
+    nodes_in_subnet: u32,
```

Comment:
nit: I would use `num_nodes_in_subnet` here and everywhere to be more explicit.

```suggestion
    num_nodes_in_subnet: u32,
```